### ISSUE TITLE
Tootlip enable

### ISSUE THREAD

**Issue by mrzepinski** [ng-bootstrap/ng-bootstrap#1342]
Is there any property to set tooltip should be shown or not?
Angular-Bootstrap for AngularJS 1.x has something like `tooltip-enable` property: https://angular-ui.github.io/bootstrap/#!#tooltip

**Comment by jnizet** [comment]
No, but you can simply set the tolltip text to null to prevent it from appearing. See http://plnkr.co/edit/qBPrlnz3ttfExCuolvFc?p=preview for example.

**Comment by jnizet** [comment]
And actually, any falsy value can do, so it can be simplified to

    [ngbTooltip]="tooltipsEnabled && 'Tooltip on top'"

http://plnkr.co/edit/npqI0IOvPu6pru8qLxN1?p=preview
